brainstormforce/astra
# AST-1195 - Mobile menu does not close when you click a one page navigation link inside them

### Description
- If any menu-item has # link, then that was not closing after clicking it, 
- due to that the dropdown & offcanvas open

### Screenshots
- https://bit.ly/2XNh0l2

### Types of changes
- Bug fix 

### How has this been tested?
- Check as per steps mentioned in tasks
- Apart from that check dropdown target option setting - https://share.bsf.io/E0uKY9WO
- Check all offcanvas styles

### Checklist:
